lib/diagnostic.rb

```diff
@@ -115,10 +117,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = '{:age => 27, :first_name => "Tina"}'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person[:first_name] << " is nice."'
```

Comment:
No need to shovel the string into it. 

`Diagnostic.hash_default = 'person.default = "#{person[:first_name]} is nice"